riff-raff.yaml

```diff
@@ -2,6 +2,12 @@ stacks: [mobile]
 regions: [eu-west-1]
 
 deployments:
+  football-time-machine:
+    type: cloud-formation
+    parameters:
+      templatePath: cfn.yaml
```

Comment:
My understanding is that we deploy `football-time-machine` to `CODE` only.  Do you think it would be good to use [`templateStagePaths`](https://riffraff.gutools.co.uk/docs/magenta-lib/types#cloudformation) here to explicitly set the stage to `CODE`?

Reply:
Oh yeah that's a good idea, thanks